src/components/ImportColumn.tsx

```diff
@@ -0,0 +1,182 @@
+import {Str} from 'expensify-common';
+import React, {useEffect} from 'react';
+import {View} from 'react-native';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {setColumnName} from '@libs/actions/ImportSpreadsheet';
+import CONST from '@src/CONST';
+import ButtonWithDropdownMenu from './ButtonWithDropdownMenu';
+import Text from './Text';
+
+const findColumnName = (header: string) => {
```

Comment:
Let's convert to a named function and add a return type 👌

Reply:
✅